Source/Pawnmorphs/Esoteria/HPatches/GeneTrackerPatches.cs

```diff
@@ -0,0 +1,29 @@
+﻿using HarmonyLib;
+using Pawnmorph.GraphicSys;
+using RimWorld;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Verse;
+
+namespace Pawnmorph.HPatches
+{
+	[HarmonyPatch(typeof(Pawn_GeneTracker))]
+	static class GeneTrackerPatches
+	{
+		[HarmonyPatch("EnsureCorrectSkinColorOverride"), HarmonyPostfix]
+		static void EnsureCorrectSkinColorOverridePostfix(Pawn ___pawn)
+		{
+			GraphicsUpdaterComp graphicsComp = ___pawn.TryGetComp<GraphicsUpdaterComp>();
+			if (graphicsComp != null)
+			{
+				graphicsComp.GeneOverrideColor = ___pawn.story.skinColorOverride;
+				graphicsComp.IsDirty = true;
+				graphicsComp.CompTick();
```

Comment:
I haven't looked at the definition of GraphicsUpdaterComp.CompTick() so this might be fine, but are you sure this is safe?  I know that manually calling Tick() can occasionally cause weird bugs with things that assume they are only ticked by the game tick (e.g. hediffs advancing quicker than they should).  Granted, a graphics comp is not very likely to do something that could cause any issues, but I figured I'd ask anyway.

Reply:
Though maybe adding a public update method would make it more intuitive.

Reply:
I figured as much!  Just felt like I had to ask just in case it was doing anything more.